app/classifier/tasks/dropdown/dropdown-dialog.cjsx

```diff
@@ -219,9 +251,25 @@ DropdownDialog = React.createClass
           render={@renderOption}
           onChange={@onReorder}
         />
+      </div>
 
-        <input ref="optionInput"></input>{' '}
-        <button type="button" onClick={@onClickAddOption}><i className="fa fa-plus"/> Add Option</button>
+      <div>
+        <p>
+          <span className="form-label">Add Option(s){' '}</span>
+          <TriggeredModalForm trigger={
+            <span className="secret-button">
+              <i className="fa fa-question-circle"></i>
+            </span>
+          }>{dropdownEditorHelp.options}
+          </TriggeredModalForm>
+        </p>
+        {if @checkConditions()
```

Comment:
1. should I include the presets area within checkCondition area so also disabled unless Options For selected?
2. if yes to #1, should delete lines 125-126?